Testnet release plan.md

```diff
@@ -0,0 +1,55 @@
+# Testnet versions
+
+This is the large task breakdown I propose we add to the next versions of testnet.
+There will be bug fixes and UX improvements as we discover them, which will be treated as high priority.
+
+Note that there is no protocol work listed here. These are the tasks that are orthogonal to the protocol.
+
+
+## V2
```

Comment:
A pedantic point but are we going to call these versions 2, 3, 4, 5 etc? At the moment we are on version 0.3.0 - the intention being AFAIK not moving to version 1.x until mainnet etc.

Reply:
I viewed these versions as conceptual milestones.
The exact releases will probably be different, based on bugs, the pace of development of the different components, etc

Reply:
Sounds good